### DISCUSSION TITLE
VrnetDialog as bus interface

### DISCUSSION THREAD

**Discussion by anline380** [john30/ebusd#534] [Q&A]
Maybe someone knows if the Vaillant VrnetDialog(860/2) communication module can be used as an interface for ebusd. The X80 port (via Raspberry RS232 Rj12 USB) is shown in the documentation as a bus connection port. It would certainly not only be interesting for me, since some of these modules are probably lying around uselessly due to the switched off service and I would otherwise dispose of them. After starting ebusd (HomeAssistant supervised add-on) for a certain time, the LEDs start flashing red and green. After bus started with own address 31/36, there is no further output on the bus side. Maybe someone has expertise, thanks Manuel 
![vrnet](https://user-images.githubusercontent.com/68514053/154049700-1defca0c-a287-48aa-b9e3-7c194959e8b6.png)
<img width="1037" alt="ebusd_log" src="https://user-images.githubusercontent.com/68514053/154049984-58227c77-e1c3-42c9-a05c-22041712b2c7.png">

**Comment by john30** [comment]
don't think this is a RS232 port, but rather eBUS directly. if so, you'd have to have an adapter anyway
